tests/ZiraLink.UnitTests/Tools/TestTools.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Microsoft.EntityFrameworkCore;
+
+using ZiraLink.Api.Application;
+using ZiraLink.Domain;
+using ZiraLink.Domain.Enums;
+namespace ZiraLink.UnitTests.Tools
+{
+    public class TestTools
+    {
+
+        public static DbContextOptions<AppDbContext>? _contextOptions;
+        public static AppDbContext? _dbContext;
+
+        /// <summary>
+        /// Initialization
+        /// </summary>
+        public static void Initialize()
+        {
+            var dbContextOptionsBuilder = new DbContextOptionsBuilder<AppDbContext>();
+            dbContextOptionsBuilder.UseInMemoryDatabase("AppDbContext");
+            _contextOptions = dbContextOptionsBuilder.Options;
+            _dbContext = new AppDbContext(_contextOptions);
+            SeedData();
+        }
+
+        /// <summary>
+        /// Initializing new data
+        /// </summary>
+        public static void SeedData()
+        {
+            if (_dbContext is not null)
```

Comment:
DbContext should not be nullable, this line will be removed

Reply:
Done.